src/ucs/config/global_opts.c

```diff
@@ -285,6 +296,12 @@ static ucs_config_field_t ucs_global_opts_read_only_table[] = {
   "Maximal size of profiling log. New records will replace old records.",
   ucs_offsetof(ucs_global_opts_t, profile_log_size), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"RCACHE_THRESHOLDS", "4k,32k,256k,1m",
```

Comment:
why not just show by power of 2?

Reply:
it was a requirement to make it configurable

Reply:
RCACHE_STAT_MIN
RCACHE_STAT_MAX
- maybe round them to power-of-2
- count power-of-2 in between min..max